### ISSUE TITLE
Unable to reproduce the results of the paper.

### ISSUE THREAD

**Issue by fftao294** [CASIA-IVA-Lab/FiLo#6]
The results cannot be reproduced according to the settings of the paper. What could be the reason for this?
![屏幕截图 2024-10-23 165621](https://github.com/user-attachments/assets/f5080066-b35d-4c7e-9e41-367164306603)
![屏幕截图 2024-10-23 165551](https://github.com/user-attachments/assets/c3336221-a14d-471e-94d4-49eec006bc31)

**Comment by FantasticGNU** [comment]
Due to differences in library versions, CUDA versions, and random seeds used in each run, as well as factors like stochastic gradient descent during training, the results may vary slightly each time. You can test the trained checkpoints we provide to reproduce the results reported in the paper.

**Comment by jiamfeng** [comment]
I used the checkpoints in the link, but the results is more lower than the paper, I can't find the reason? can you help me to analyse?

![Image](https://github.com/user-attachments/assets/cd303146-ddc4-46a3-99d8-a1de1968dd70)

**Comment by FantasticGNU** [comment]
In MVTec AD dataset, the results in our paper is 91.2 Image AUC and 92.3 Pixel AUC, the results you provide is even higher than the paper.

**Comment by jiamfeng** [comment]
> In MVTec AD dataset, the results in our paper is 91.2 Image AUC and 92.3 Pixel AUC, the results you provide is even higher than the paper.

Yes，the performance is higher，but when I visualizer the result, it still have more false positive, it's normal ? 
code is below. (if it's uncorrect, please tell me, thanks) 
![Image](https://github.com/user-attachments/assets/21e0a2ac-eddc-4562-9d61-d5772dd464e7)

![Image](https://github.com/user-attachments/assets/26aa8618-1538-4811-ae2e-6aab2499e12a)

![Image](https://github.com/user-attachments/assets/2215102e-cb59-4d6d-b31f-cea4bd0833b0)


One more question.
1. When I read the codes, I confused that the GroundingDINO only participates in inference and does not participate in training ? 
In my view, if it participate in training maybe can filter more false positive and get better results ? maybe my opinion is wrong。
2. what's the steps to train dataset in GroundingDINO ?

Best Regards !
